### ISSUE TITLE
Some Deltaboard related changes

### ISSUE THREAD

**Issue by Snorrrlax** [MystK/delta-bot-three#30]
Thanks for setting up Deltaboards! Very exciting. I just noticed a few things:

1) Could we move the sidebar Deltaboard so it's between "The Delta System" and "Wiki and Mod Standards"

2) Can the title in the sidebar be changed from "Monthly Deltaboards" to "Monthly Deltaboard"

3) I noticed you included a "last updated" timer on the sidebar, that's really cool. But can this be changed so it only updates the time when something on the Deltaboard has actually changed? I've seen the time update a few times now but I don't think any of the scores have changed. 

Can you also include the timezone used?

4) Can we change all the deltaboard layouts to this:

| Rank | Username | Deltas |
| :------: | :------: | :------: |
| 1 | **garnteller** ([181&#8710;](https://www.reddit.com/r/changemyview/wiki/user/garnteller "garnteller's delta history wiki page")) | 9 |
| 2 | Ansuz07 ([181&#8710;](https://www.reddit.com/r/changemyview/wiki/user/garnteller "garnteller's delta history wiki page")) | 9 |
| 3 | cacheflow ([181&#8710;](https://www.reddit.com/r/changemyview/wiki/user/garnteller "garnteller's delta history wiki page")) | 7 |
| 4 | srnkmrsn ([181&#8710;](https://www.reddit.com/r/changemyview/wiki/user/garnteller "garnteller's delta history wiki page")) | 6 |
| 5 | Gladix ([181&#8710;](https://www.reddit.com/r/changemyview/wiki/user/garnteller"garnteller's delta history wiki page" )) | 5 |
| 6 | Amablue ([181&#8710;](https://www.reddit.com/r/changemyview/wiki/user/garnteller "garnteller's delta history wiki page")) | 5 |
| 7 | cdb03b ([181&#8710;](https://www.reddit.com/r/changemyview/wiki/user/garnteller "garnteller's delta history wiki page")) | 5 |
| 8 | PreacherJudge ([181&#8710;](https://www.reddit.com/r/changemyview/wiki/user/garnteller "garnteller's delta history wiki page")) | 5 |
| 9 | Grunt08 ([181&#8710;](https://www.reddit.com/r/changemyview/wiki/user/garnteller "garnteller's delta history wiki page")) | 4 |
| 10 | Generic_On_Reddit ([181&#8710;](https://www.reddit.com/r/changemyview/wiki/user/garnteller "garnteller's delta history wiki page")) | 4 |

To sum up the changes here, we drop the **/u/** from the usernames, include a wiki link in brackets with their total delta score - which when hovered over says "username's delta history wiki page" (I just copied and pasted garnteller's one here), and drop the delta symbol and link from the third column. 

5) Underneath the last updated note in the sidebar, can we include a link that says something along the lines of "[See our other Deltaboards in the wiki!](https://www.reddit.com/r/changemyview/wiki/deltaboards)"

Thanks!

**Comment by Snorrrlax** [comment]
Also,

6) Can all the text in the Deltaboards be centred. This is the current markup:

| :------: | ------ | :------: |

But it should be:

| :------: | :------: | :------: |

https://www.reddit.com/wiki/commenting#wiki_tables

**Comment by Snorrrlax** [comment]
I just spoke to /u/qtx (creator of our CSS) about how to stylistically fit in that "last updated" line that currently sticks out a bit under the sidebar. This was his response:

Easiest way and without needing to hack something together is to just add this right after the table on the sidebar:


    | |updated 9/25/2016, 2:11:26 PM| |

That way it will look part of the table, you can see how it looks on /r/cmvcss2015/
Be sure to put it right under the table, so no separate line between them. Like so:

    | Rank | Username | Deltas |
    | :------: | :------: | :------: |
    | 1 | **/u/garnteller** | [10∆](// "deltas received") |
    | 2 | /u/Ansuz07 | [9∆](// "deltas received") |
    | 3 | /u/cacheflow | [7∆](// "deltas received") |
    | 4 | /u/srnkmrsn | [6∆](// "deltas received") |
    | 5 | /u/Gladix | [5∆](// "deltas received") |
    | 6 | /u/Amablue | [5∆](// "deltas received") |
    | 7 | /u/cdb03b | [5∆](// "deltas received") |
    | 8 | /u/PreacherJudge | [5∆](// "deltas received") |
    | 9 | /u/Grunt08 | [4∆](// "deltas received") |
    | 10 | /u/Generic_On_Reddit | [4∆](// "deltas received") |
    | |updated 9/25/2016, 2:11:26 PM| |

I removed 'last' from the sentence to make it fit, hope that's ok :)

Oh and btw, notice the space between the two |'s, that's important, so don't leave them out =)

**Comment by MystK** [comment]
Howdy I made some changes. I implemented, 1, 2, partial 3, partial 4, and 5.

For 3, I changed it from updated to checked. It's an indicator of when DB3 last checked so we can see if it crashed or something. Did we want to keep that? If not, I can change it to last updated, or both. Last updated to also take some more time. Once we figure out what we want to do, I'll make a separate issue.

For 4, I need to add a call to grab the total delta count, So it will take some time. I made the username the link to the delta wiki ATM. In the meantime, did you want me to change up the table to something else? I will split the final table into a separate issue. Hopefully someone can work on it.

**Comment by Snorrrlax** [comment]
Thanks!

I'm not sure marking when it's last checked is a good idea since that's a lot of mod actions? I think it would be more useful information to everyone (not just mods) if it said when it last made an update rather than a check? (In the Deltaboards wiki page as well)

I'm assuming we'd notice it had crash in other ways rather than by checking this timer.

Can you also include the link to the wiki page as a row in the table like you did with the other line? Ideally we want to make both of them fit in one line, any suggestions?

**Comment by MystK** [comment]
For the date, I could do

`9/25/2016 15:36:40 EDT`

Is that good?

In the meantime, I'll remove the date because it is last checked not last updated

**Comment by Snorrrlax** [comment]
Yeah that would be good, thanks!

**Comment by Snorrrlax** [comment]
Actually, if you knocked off the seconds you could do: 

As of 9/25/16 15:36 EDT

And then in the row below: 

[More Deltaboards](http://www.reddit.com/r/changemyview/wiki/deltaboards)

**Comment by Snorrrlax** [comment]
So just to summarise, we'll delete "[See our other Deltaboards in the wiki!](https://www.reddit.com/r/changemyview/wiki/deltaboards)"

And add two lines under the table:

| Rank | Username | Deltas |
| :------: | :------: | :------: |
| 1 | **/u/garnteller** | [10∆](// "deltas received") |
| 2 | /u/Ansuz07 | [9∆](// "deltas received") |
| 3 | /u/cacheflow | [7∆](// "deltas received") |
| 4 | /u/srnkmrsn | [6∆](// "deltas received") |
| 5 | /u/Gladix | [5∆](// "deltas received") |
| 6 | /u/Amablue | [5∆](// "deltas received") |
| 7 | /u/cdb03b | [5∆](// "deltas received") |
| 8 | /u/PreacherJudge | [5∆](// "deltas received") |
| 9 | /u/Grunt08 | [4∆](// "deltas received") |
| 10 | /u/Generic_On_Reddit | [4∆](// "deltas received") |
| |As of 9/25/16 15:36 EDT
| |
| |[More Deltaboards](http://www.reddit.com/r/changemyview/wiki/deltaboards)| |

**Comment by MystK** [comment]
done

**Comment by Snorrrlax** [comment]
Thanks! 

Any idea why there's a space below the table on the CMV sidebar? 

Also what do you want to do about the partial 4 bit? It's low priority but would a separate issue help?

**Comment by MystK** [comment]
Yes, will have to separate that out to separate issue. It's pretty straightforward just have to spend some time.

for the space, I will poke around.

**Comment by MystK** [comment]
removed the space
